docs/architecture/gas/parameter_definition.md

```diff
@@ -10,7 +10,27 @@ starting from version 53. To see all parameter values for a specific version at
 once, check out the list of JSON snapshots generated in this directory:
 [core/primitives/src/runtime/snapshots](https://github.com/near/nearcore/blob/d0dc37bf81f7e7bde9c560403b085fae04108659/core/primitives/src/runtime/snapshots).
 
-<!-- TODO: Runtime Config Store -->
+## Using Parameters in Code
+
+The introduction on this page already hints at it. Parameter values are
+versioned. In other words, they can change between protocol versions. A nearcore
+binary has to support multiple versions and choose the correct parameter value at
+runtime.
+
+To make this easy, there is
+[`RuntimeConfigStore`](https://github.com/near/nearcore/blob/a8964d200b3938a63d389263bc39c1bcd75b1de4/core/primitives/src/runtime/config_store.rs#L43).
+It contains a sparse map from protocol versions to complete runtime
+configurations (`BTreeMap<ProtocolVersion, Arc<RuntimeConfig>>`).
+The runtime then uses `store.get_config(protocol_version)` to access a runtime
```

Comment:
```suggestion
The runtime then uses `store.get_config(protocol_version)` to access the runtime
```

Reply:
Hm, it's one runtime config of many in the list, so I think `a` is also correct here and it puts more emphasis on the fact that there are multiple configs to choose from.

Reply:
Ah you are right then.